app/views/partials/shared/nav/get-started.liquid

```diff
@@ -24,8 +24,8 @@
   </ul>
 </li>
 
-<li class="mt-6 font-bold text-lg">
-  Build your first app
+<li class="mt-6 mb-2 text-base topic-group">
+  <h3 class="text-base">BUILD YOUR FIRST APP</h3>
```

Comment:
There is an `uppercase` class that you could use instead of manually capitalise the whole sentence.